src/Transfer/EzPlatform/Repository/Manager/LanguageManager.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of Transfer.
+ *
+ * For the full copyright and license information, please view the LICENSE file located
+ * in the root directory.
+ */
+
+namespace Transfer\EzPlatform\Repository\Manager;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\LanguageService;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\Values\Content\Language;
+use eZ\Publish\API\Repository\Values\Content\LanguageCreateStruct;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+use Transfer\EzPlatform\Data\LanguageObject;
+
+/**
+ * Content type manager.
+ *
+ * @author Harald Tollefsen <harald@netmaking.no>
+ */
+class LanguageManager implements LoggerAwareInterface
```

Comment:
I think we should have those classes implement the right interfaces - for the sake of uniformity in relation to other similar classes.